izanami-frontend/src/pages/users.tsx

```diff
@@ -90,6 +89,104 @@ export function Users() {
       createInvitation(data.email, data.admin, data.rights)
   );
 
+  function OperationToggleForm(props: {
+    bulkOperation: string;
+    selectedRows: UserType[];
+    cancel: () => void;
+  }) {
+    const { bulkOperation, selectedRows, cancel } = props;
+    switch (bulkOperation) {
+      case "Toggle Admin Role": {
+        const [values, setSelectedValues] = React.useState<Option[] | null>();
+        const onSelected = (selectedOptions: Option[]) => {
```

Comment:
As we discussed before, these one line function that are used only in one sport should be inlined